src/components/ResultContainer.tsx

```diff
@@ -144,147 +144,103 @@ function ResultContainer({
-                  'pipeline',
-                  parsePipelinesInfoToString(res.available_pipelines).split(', ')
-                ),
-              ].join('\t')
-            );
-          } else {
-            // @ts-expect-error: typescript doesn't know that subject_data is an array when records_protected is false.
-            res.subject_data.forEach((subject) => {
-              tsvRows.push(
-                [
-                  res.dataset_name.replace('\n', ' '),
-                  res.dataset_portal_uri,
-                  res.num_matching_subjects,
-                  subject.sub_id,
-                  subject.session_id,
-                  subject.session_file_path,
-                  convertURIToLabel('sessionType', subject.session_type),
-                  subject.age,
-                  convertURIToLabel('sex', subject.sex),
-                  convertURIToLabel('diagnosis', subject.diagnosis),
-                  convertURIToLabel('assessment', subject.assessment),
-                  subject.num_matching_phenotypic_sessions,
-                  subject.num_matching_imaging_sessions,
-                  convertURIToLabel('modality', subject.image_modal),
-                  convertURIToLabel(
-                    'pipeline',
-                    parsePipelinesInfoToString(subject.completed_pipelines).split(', ')
-                  ),
-                  convertURIToLabel('modality', res.image_modals),
-                  convertURIToLabel(
+      const headers = [
+        'DatasetName',
+        'PortalURI',
+        'NumMatchingSubjects',
+        'SubjectID',
+        'SessionID',
+        'ImagingSessionPath',
+        'SessionType',
+        'NumMatchingPhenotypicSessions',
+        'NumMatchingImagingSessions',
+        'Age',
+        'Sex',
+        'Diagnosis',
+        'Assessment',
+        'SessionImagingModalities',
+        'SessionCompletedPipelines',
+        'DatasetImagingModalities',
+        'DatasetPipelines',
+      ].join('\t');
+      tsvRows.push(headers);
+      datasets.forEach((res) => {
+        if (res.records_protected) {
+          tsvRows.push(
+            [
+              res.dataset_name.replace('\n', ' '),
+              res.dataset_portal_uri,
+              res.num_matching_subjects,
+              'protected', // subject_id
+              'protected', // session_id
+              'protected', // session_file_path
+              'protected', // session_type
+              'protected', // num_matching_phenotypic_sessions
+              'protected', // num_matching_imaging_sessions
+              'protected', // age
+              'protected', // sex
+              'protected', // diagnosis
+              'protected', // assessment
+              'protected', // session_imaging_modality
+              'protected', // session_completed_pipelines
+              isHumanFile
+                ? convertURIToLabel('modality', res.image_modals)
+                : res.image_modals?.join(', '),
```

Comment:
I think the decision was to not have spaces in the comma separated arrays in the table. I.e. in line 194 (old 222) below we would also like to remove the `.split(', ') - if I understand this correctly